indexing-service/src/main/java/org/apache/druid/indexing/common/task/HadoopIndexTask.java

```diff
@@ -218,10 +223,20 @@ public String getClasspathPrefix()
     return classpathPrefix;
   }
 
+  public String getHadoopJobIdFileName()
+  {
+    final String HADOOP_JOB_ID_FILENAME = "mapReduceJobId.json";
+    StringBuilder fileName = new StringBuilder(taskConfig.getTaskDir(getId()).toString());
+    fileName.append("/");
+    fileName.append(HADOOP_JOB_ID_FILENAME);
+    return fileName.toString();
```

Comment:
Should be `return new File(taskConfig.getTaskDir(getId()), HADOOP_JOB_ID_FILENAME).getAbsolutePath();`.